<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>691</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>691</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>531</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="GNSS_lineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="INS_lineEdit_2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="OutPut_lineEdit_3"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>61</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>GNSS：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>INS：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>OutPut：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>82</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="GNSS_Input">
       <property name="text">
        <string>选择</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/Picture/Select.png</normaloff>:/Picture/Select.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="INS_Input">
       <property name="text">
        <string>选择</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/Picture/Select.png</normaloff>:/Picture/Select.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="GINS_Output">
       <property name="text">
        <string>选择</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/Picture/Select.png</normaloff>:/Picture/Select.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>691</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="GINS_Caculate">
       <property name="text">
        <string>解算</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/Picture/Star.png</normaloff>:/Picture/Star.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="GINS_Config">
       <property name="text">
        <string>配置</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/Picture/Config.png</normaloff>:/Picture/Config.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="GINS_Plot">
       <property name="text">
        <string>绘图</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/Picture/Plot.png</normaloff>:/Picture/Plot.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="GINS_Close">
       <property name="text">
        <string>关闭</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/Picture/Exit.png</normaloff>:/Picture/Exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>691</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/Picture/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件操作(&amp;F)</string>
    </property>
    <addaction name="action_Config"/>
    <addaction name="action_POS"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>结果绘图(&amp;P)</string>
    </property>
    <addaction name="action_Draw_result"/>
    <addaction name="action_Draw_error"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>软件信息(&amp;I)</string>
    </property>
    <addaction name="action_Information"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Config">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Picture/Open.png</normaloff>:/Picture/Open.png</iconset>
   </property>
   <property name="text">
    <string>导入Config文件</string>
   </property>
  </action>
  <action name="action_POS">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Picture/Save.png</normaloff>:/Picture/Save.png</iconset>
   </property>
   <property name="text">
    <string>保存POS文件</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Picture/Exit.png</normaloff>:/Picture/Exit.png</iconset>
   </property>
   <property name="text">
    <string>关闭软件</string>
   </property>
  </action>
  <action name="action_Draw_result">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Picture/Result_Plot.png</normaloff>:/Picture/Result_Plot.png</iconset>
   </property>
   <property name="text">
    <string>定位结果绘制</string>
   </property>
  </action>
  <action name="action_Draw_error">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Picture/Plot_Error.png</normaloff>:/Picture/Plot_Error.png</iconset>
   </property>
   <property name="text">
    <string>误差绘制</string>
   </property>
  </action>
  <action name="action_Information">
   <property name="text">
    <string>软件信息</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
